.github/workflows/release-source.yml

```diff
@@ -93,7 +93,6 @@ jobs:
       run: |
```

Comment:
Mind adding a `shell: bash` here? 

Turns out, by default GitHub doesn't always cancel the step if any of the subprocesses exit non-zero (especially in a pipe). And we really want that to happen here. With `shell: bash` this is done correctly. Should make no difference for your work, just a nice "while we are here" :)